hw1/02-palindrome.js

```diff
@@ -1,3 +1,28 @@
-const elem = document.querySelector('input');
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
I really like this way to check a palindrome. I've learned from you. 
Since it's a short code, it won't take too long to understand what you did. A little node I would say that you should leave a few lines of comments on what you did. That would be great!